zhaquirks/tuya/ts0601_illuminance.py

```diff
@@ -0,0 +1,115 @@
+"""Tuya HOME-LUX illuminance sensor."""
+
+import math
+from typing import Dict
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import Basic, GreenPowerProxy, Groups, Ota, Scenes, Time
+from zigpy.zcl.clusters.measurement import IlluminanceMeasurement
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.tuya import DPToAttributeMapping, TuyaLocalCluster, TuyaNewManufCluster
+
+TUYA_BRIGHTNESS_LEVEL_ATTR = 0x01  # 0-2 "Low, Medium, High"
+TUYA_ILLUMINANCE_ATTR = 0x02  # [0, 0, 3, 232] illuminance
+
+TUYA_BRIGHTNESS_LEVEL_STR = ["Low", "Medium", "High"]
+
+
+class TuyaIlluminanceMeasurement(IlluminanceMeasurement, TuyaLocalCluster):
+    """Tuya local IlluminanceMeasurement cluster."""
+
+    attributes = IlluminanceMeasurement.attributes.copy()
+    attributes.update(
+        {
+            0xFF00: ("brightness_level", t.CharacterString, True),
```

Comment:
The illuminanceMeasurement cluster don't have any reference to `brightness_level` so not sure if that could be the correct attribute name 🤔 
We can keep the reference for now but I would replace for `manufacturer_brightness_level` or `manufacturer_measured_value`.